providers/curation/github.js

```diff
@@ -304,19 +327,21 @@ class GitHubCurationService {
     return coordinates.toString()
   }
 
-  _getBranchName(coordinates) {
-    const c = coordinates
-    return `${c.type.toLowerCase()}_${c.name}_${c.revision}_${moment().format('YYMMDD_HHmmss.SSS')}`
+  async _getBranchName(userGithub, serviceGithub) {
+    const committer = userGithub ? await userGithub.users.get({}) : await serviceGithub.users.get({})
+    return `${committer.data.name.toLowerCase().replace(/ /g, "_")}_${moment().format('YYMMDD_HHmmss.SSS')}`
```

Comment:
Interesting problem. Not sure that all chars in a GitHub user name are valid as a branch name. Should think about this strategy some. The same is technically true of package names (the previous approach) but seemingly less likely. Names are all over the place with people using unicode and emojis and ...